sdk/keyvault/azure-security-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/KeyClientTestBase.java

```diff
@@ -118,6 +119,22 @@ void setKeyRunner(Consumer<CreateKeyOptions> testRunner) {
         testRunner.accept(keyOptions);
     }
 
+    @Test
+    public abstract void createRsaKey(HttpClient httpClient, KeyServiceVersion keyServiceVersion);
+
+    void createRsaKeyRunner(Consumer<CreateRsaKeyOptions> testRunner) {
+        final Map<String, String> tags = new HashMap<>();
+
+        tags.put("foo", "baz");
+
+        final CreateRsaKeyOptions createRsaKeyOptions = new CreateRsaKeyOptions(KEY_NAME)
```

Comment:
We should use `generateResourceId()` for the key name since it generates a randomized value, which helps when running live tests.
```suggestion
        final CreateRsaKeyOptions createRsaKeyOptions = new CreateRsaKeyOptions(generateResourceId(KEY_NAME))
```